<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="example-test"
    errors="1" failures="0" hostname="" id="" package="example-test" skipped="0"
    tests="1" time="19.98" timestamp="Thu, 29 Nov 2018 20:59:41 GMT">
  
    <testcase name="Nightwatch Example" classname="example-test" time="19.98" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Example Test / Nightwatch Example]:

  AssertionError [ERR_ASSERTION]: true == false
    at /Users/justin/repos/applitools/js/Nightwatch/tests/example-test.js:41:20
    at ManagedPromise.invokeCallback_ (/Users/justin/repos/applitools/js/Nightwatch/node_modules/selenium-webdriver/lib/promise.js:1376:14)
    at TaskQueue.execute_ (/Users/justin/repos/applitools/js/Nightwatch/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/Users/justin/repos/applitools/js/Nightwatch/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/Users/justin/repos/applitools/js/Nightwatch/node_modules/selenium-webdriver/lib/promise.js:2927:27)
    at /Users/justin/repos/applitools/js/Nightwatch/node_modules/selenium-webdriver/lib/promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)
    </system-err>
  

  
  </testsuite>
</testsuites>
